invenio_oauthclient/views/client.py

```diff
@@ -126,7 +126,12 @@ def authorized(remote_app=None):
            not(current_app.debug or current_app.testing)):
             abort(403)
 
-    return current_oauthclient.handlers[remote_app]()
+    try:
+        handler = current_oauthclient.handlers[remote_app]()
+    except RuntimeError:
```

Comment:
Yes, for some magic reason when I was checking with OAuthException it wasn't working..Now it's ok and amended..

BTW, the response we are getting from CERN OAuth is ```{u'error': u'invalid_grant'}```, so it might need further investigation from our side and the CERN OAuth side